packages/shared/src/sdk-versioning/sdk-versions/php.json

```diff
@@ -0,0 +1,8 @@
+{
+  "versions": [
+    {
+      "version": "1.2.0",
+      "capabilities": ["bucketingV2", "encryption"]
+    }
```

Comment:
Add `looseUnmarshalling` to either version `0` or `1.0.0` depending on how we interpret it.  Prior to 1.0.0, we would log a warning on unknown keys, but not stop execution.